examples/video/videoGrabberExample/src/ofApp.cpp

```diff
@@ -2,123 +2,106 @@
 
 //--------------------------------------------------------------
 void ofApp::setup(){
-	
-	camWidth 		= 320;	// try to grab at this size. 
-	camHeight 		= 240;
-	
-    //we can now get back a list of devices. 
-	vector<ofVideoDevice> devices = vidGrabber.listDevices();
-	
+    camWidth = 320;  // try to grab at this size.
+    camHeight = 240;
+
+    //we can now get back a list of devices.
+    vector<ofVideoDevice> devices = vidGrabber.listDevices();
+
     for(int i = 0; i < devices.size(); i++){
-		cout << devices[i].id << ": " << devices[i].deviceName; 
-        if( devices[i].bAvailable ){
+        cout << devices[i].id << ": " << devices[i].deviceName;
```

Comment:
I realize this was already here, but seems odd that we still have a `std::cout` in the code.  Wouldn't it be better to wrap it up in an `ofLog*`?